src/theme/lightTheme.js

```diff
@@ -1,9 +1,11 @@
 const lightTheme = {
   "color-primary": "#2970ff",
-  "color-primary-dark": "#091440",
+  "color-primary-dark": "var(--color-blue-darkest)",
```

Comment:
Hrmm don't change theme variables values this affects so many other places also in `pigui`.

I guess you want to create new theme variables which are specific to your components needs
then use these in css styling to apply custom colors.